assets/styles/_bio.css

```diff
@@ -0,0 +1,28 @@
+.bio {
```

Comment:
I like your CSS structure. You can try leaving an empty line after each block to make it more readable.